src/ui/popup.html

```diff
@@ -37,6 +37,26 @@
     <link rel="preload" as="image" href="https://protocol-icons.s3.amazonaws.com/uniswap-exchangev3.png">
     <link rel="preload" as="image" href="https://protocol-icons.s3.amazonaws.com/paraswap.png">
 
+    <!-- DNA Mint Flow -->
```

Comment:
Another way would be to programmatically add these elements from the dna mint module, but as long as we don't forget to remove these later it's ok